buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/fixture/hadoop/InstanceInfo.groovy

```diff
@@ -236,7 +237,7 @@ class InstanceInfo {
 
     /** Return the java home used by this node. */
     String getJavaHome() {
-        return javaVersion == null ? project.runtimeJavaHome : project.javaVersions.find { it.version == javaVersion }.javaHome.absolutePath
+        return javaVersion == null ? project.runtimeJavaHome : BuildParams.javaVersions.find { it.version == javaVersion }.javaHome.absolutePath
```

Comment:
Assuming we refactor out usages of extra properties completely, we can replace `project.runtimeJavaHome` with the `BuildParams` variant.